### PR TITLE
Fix #36664

### LINKED ISSUE #36664: [Enhancement]: Add page controls at bottom of tax rates list
### Describe the solution you'd like

In WC Settings, Taxes, On the Standard Tax Rates list, it would be helpful to have the page forward/back controls at the bottom as well as at the top of screen. By the time you scroll to the bottom, if you have a lot of rates, the top has disappeared.

### Describe alternatives you've considered

_No response_

### Additional context

Background is that because I don't trust our website to properly enforce a country choice, I have the full list of 250 ISO countries in our list set explicitly rather than relying on the asterisk catchall for those we don't trade with.

### PR DESCRIPTION
**Pull request by TarekNabil**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Added another pagination to the bottom of the tax rates page

Closes #36664.

At the settings page -> Tax tab -> Standard rates -> when there is many rates, pagination will be on the bottom as well as the top.

### CODE REVIEW

File: plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js [hunk 1]
```diff
@@ -19,6 +19,7 @@
 			$tbody             = $( '#rates' ),
 			$save_button       = $( ':input[name="save"]' ),
 			$pagination        = $( '#rates-pagination' ),
+			$bottom_pagination = $( '#rates-bottom-pagination' ),
```
Comment (rrennick) [inline plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js]: @TarekNabil Thanks for the PR. Is there a reason you didn't include the new pagination element in `$pagination`?

```
$pagination = $( '#rates-pagination, #rates-bottom-pagination' )
```

Doing that would eliminate all the other JS changes.

Comment (TarekNabil) [reply plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js]: No, you are right. I should have done this.

Comment (TarekNabil) [reply plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js:22]: What do you think I should do now? Make the edits and commit again or create a new PR?

Comment (TarekNabil) [reply plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js]: Thanks for the advice, Do I need to create a new PR now?

**Review by rrennick — APPROVED** [review]
@TarekNabil Thanks for adding this. I added the changelog to the PR for you so that it could be merged.

**Pull request merged**